plugin/showmarks.vim

```diff
@@ -36,13 +36,25 @@ endif
 
 " Options: Set up some nice defaults
 if !exists('g:showmarks_enable'      ) | let g:showmarks_enable       = 1    | endif
-if !exists('g:showmarks_auto_toggle' ) | let g:showmarks_auto_toggle  = 1    | endif
 if !exists('g:showmarks_no_mappings' ) | let g:showmarks_no_mappings  = 0    | endif
 if !exists('g:showmarks_ignore_type' ) | let g:showmarks_ignore_type  = "hq" | endif
 if !exists('g:showmarks_hlline_lower') | let g:showmarks_hlline_lower = "0"  | endif
 if !exists('g:showmarks_hlline_upper') | let g:showmarks_hlline_upper = "0"  | endif
 if !exists('g:showmarks_hlline_other') | let g:showmarks_hlline_other = "0"  | endif
 
+" The value of showmarks_auto_toggle was changed to be a list. For backward
+" compatibility reasons, the code still supports it as a number or as a string.
+if !exists('g:showmarks_auto_toggle' ) | let g:showmarks_auto_toggle  = ['cursor_hold'] | endif
+if !exists('v:t_number') && type(g:showmarks_auto_toggle) == type(0)
```

Comment:
The `v:t_number` check seems redundant, and causes Vim 8.1 skip later logic.

Reply:
Yeah, it was my bad. I started by doing the condition a bit differently. When I changed the condition I forgot about this part of it.

It should work now on Vim8.1 as well.